docz/package.json

```
@@ -4,7 +4,8 @@
   "scripts": {
     "dev": "docz dev",
     "build": "docz build",
-    "now-build": "yarn build"
+    "now-build": "yarn build",
```

```suggestion
    "now-build": "docz build",
```

Let's make these consistent. And not every user is going to know about yarn.